src/Symfony/Component/Console/Application.php

```diff
@@ -61,7 +61,7 @@ class Application
     private $autoExit;
     private $definition;
     private $helperSet;
-    private $dispatcher;
+    protected $dispatcher;
```

Comment:
You can simply access it the same way as your application does: `$container->get('event_dispatcher')`